File: llm_generated_code/Gemini/CPP/Level1/Q31.cpp
#include <iostream>

using namespace std;

int main() {
    int t;
    cin >> t;

    while (t--) {
        int a, b;
        cin >> a >> b;

        if (a == b) {
            cout << "-1" << endl;
        } else if (a > b) {
            cout << "Alice" << endl;
        } else {
            cout << "Bob" << endl;
        }
    }

    return 0;
}

File: llm_generated_code/Gemini/Python/Level1/Q31.py
def winner(a, b):
    if a == 0 or b == 0:
        return "Bob" if a == 0 else "Alice"
    if a > b:
        return "Alice"
    else:
        return "Bob"

def main():
    t = int(input())
    for _ in range(t):
        a, b = map(int, input().split())
        print(winner(a, b))

if __name__ == "__main__":
    main()
